RocksmithTookitGUI/DLCPackageCreator/ArrangementForm.cs

```diff
@@ -341,6 +342,21 @@ public void LoadArrangementData(string xmlFilePath)
                 Arrangement.MasterId = masterId;
         }
 
+        private static ArrangementType DetectArrangementType(string filename)
+        {
+            using (XmlReader reader = XmlReader.Create(filename))
+            {
+                reader.MoveToContent();
+                if (reader.LocalName == "vocals")
+                    return ArrangementType.Vocal;
+                if (reader.LocalName == "showlights")
+                    return ArrangementType.ShowLight;
+
+                // Return guitar for everything else
+                return ArrangementType.Guitar;
```

Comment:
Ok this appears to work for bass arrangement too.   TODO: create Enum ArrangementType.Instrument for use in this type of instance.

Reply:
"You mean a tab like where you would pick some settings and it would launch my program" Yes exactly.  
Maybe that is not necessary (overkill) with your Font Generator because it already has a GUI.  So maybe just adding a button to launch the Font Generator would be enough.  Looking to make it easier for potential users.

Reply:
Toolkit is .Net 4.0 so if you used new 4.6 features it will not be compatible.  Though probably not an issue since it is standalone app.

Reply:
I will merge the pull and we can go from there.